backend/census_historical_migration/management/commands/run_paginated_migration.py

```diff
@@ -0,0 +1,41 @@
+from ...historic_data_loader import load_historic_data_for_year
+
+from django.core.management.base import BaseCommand
+
+import logging
+import sys
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
+
+
+class Command(BaseCommand):
+    help = """
+        Migrate from Census tables to GSAFAC tables for a given year using pagination
+        Usage:
+        manage.py run_migration
+            --year <audit year>
+            --batchSize <batch size>
```

Comment:
`batch_size` or `batchSize`. What you pick above will determine here

Reply:
`page_size`